geometry_msgs/PoseStamped[] prior_poses
---
bool success
